packages/react/src/components/header/Header.stories.tsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
```

Comment:
What do these header changes have to do with footer? Multiple files of seemingly only header

Reply:
Because this git branch is based on header navigation menu branch which is still under review (mainly waiting for Unicus).